typescript/phoenix-sdk/src/market.ts

```diff
@@ -89,7 +88,7 @@ export type L3Order = {
   priceInTicks: BN;
   side: Side;
   sizeInBaseLots: BN;
-  makerPubkey: string;
+  makerPubkey?: string;
```

Comment:
when would we not have this?

Reply:
out of abundance of precaution I thought it was better to leave this as possibly optional rather than throwing if the index lookup failed

Reply:
if this fails, doesn't that mean the on-chain state has been corrupted?

Reply:
Hm that's fair, I can just throw if there's a miss